code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -111,6 +111,7 @@
 			/obj/item/explosive/grenade = 600,
 			/obj/item/explosive/grenade/m15 = 30,
 			/obj/item/explosive/grenade/sticky = 125,
+			/obj/item/explosive/grenade/sticky/trailblazer = 100,
```

Comment:
This is my opinion, having a sticky incendiary shouldn't be more than incendiary. We will see whether xenomorphs cope and seeth hard.

Reply:
this was heavily based on stickies, so I reduced it a bit but yeah, if this gets TMed or merged. if 100 is too many I'll bump it down to incendiary level or maybe lower if its really bad.

Reply:
Reduced a bit more to 75 to be a good middle ground for testing.